x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -137,6 +142,95 @@ function UserDetailsEditor({ user }: { user: AuthenticatedUser }) {
   );
 }
 
+function UserSettingsEditor({ formik }: { formik: ReturnType<typeof useUserProfileForm> }) {
+  if (!formik.values.data) {
+    return null;
+  }
+
+  return (
+    <EuiDescribedFormGroup
+      fullWidth
+      fieldFlexItemProps={{ style: { alignSelf: 'flex-start' } }}
+      title={
+        <h2>
+          <FormattedMessage
+            id="xpack.security.accountManagement.userProfile.userSettingsTitle"
+            defaultMessage="User Settings"
+          />
+        </h2>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.security.accountManagement.userProfile.themeFormGroupDescription"
+          defaultMessage="Choose your desired theme for Kibana"
+        />
+      }
+    >
+      <EuiFlexGroup responsive={false}>
+        <EuiFlexItem>
+          <FormRow
+            name="data.userSettings.darkMode"
+            label={
+              <FormLabel for="data.userSettings.darkMode">
+                <FormattedMessage
+                  id="xpack.security.accountManagement.userProfile.userSettings.theme"
+                  defaultMessage="Theme"
+                />
+              </FormLabel>
+            }
+            fullWidth
+          >
+            <EuiButtonGroup
+              legend={i18n.translate(
+                'xpack.security.accountManagement.userProfile.userSettings.themeGroupDescription',
+                {
+                  defaultMessage: 'Kibana Theme',
+                }
+              )}
+              buttonSize="m"
+              data-test-subj="darkModeButton"
+              idSelected={formik.values.data.userSettings.darkMode}
+              options={[
+                {
+                  id: 'dark',
+                  label: (
+                    <FormattedMessage
+                      id="xpack.security.accountManagement.userProfile.darkModeButton"
+                      defaultMessage="Dark Mode"
+                    />
+                  ),
+                  iconType: 'moon',
+                },
+                {
+                  id: 'light',
+                  label: (
+                    <FormattedMessage
+                      id="xpack.security.accountManagement.userProfile.lightModeButton"
+                      defaultMessage="Light Mode"
+                    />
+                  ),
+                  iconType: 'sun',
+                },
+                {
+                  id: '',
+                  label: (
+                    <FormattedMessage
+                      id="xpack.security.accountManagement.userProfile.defaultModeButton"
+                      defaultMessage="Default"
```

Comment:
Am I correct in assuming this selection means the user wishes to use the current space defaults? Previously, we had discussed the possibility of removing the space-level advanced setting for dark mode entirely and replacing it with these individual user-level controls. Is that no longer being considered, or is that a future consideration? CCing @legrego.

That said, if we do plan to continue offering the space-level advanced setting for the time being, would it be better to reword this to make it clear that "Default" is in reference to the space setting?

```suggestion
                      defaultMessage="Space default"
```

Reply:
I haven't heard either way on the Spaces Adv. Setting.

'Space default' change added latest commit!